File: java-source/src/main/java/net/corda/accord/BraidServer.java
package net.corda.accord;

import io.bluebank.braid.corda.BraidConfig;
import io.vertx.core.http.HttpServerOptions;
import net.corda.accord.flow.PromissoryNoteIssueFlow;
import net.corda.core.node.AppServiceHub;
import net.corda.core.node.ServiceHub;
import net.corda.core.node.services.CordaService;
import net.corda.core.serialization.SingletonSerializeAsToken;

@CordaService
public class BraidServer extends SingletonSerializeAsToken {

    private AppServiceHub appServiceHub;

    public BraidServer(AppServiceHub serviceHub) {
        this.appServiceHub = serviceHub;

        new BraidConfig()
                .withFlow("PromissoryNoteIssueFlow", PromissoryNoteIssueFlow.InitiatorFlow.class)
                .withService("PromissoryNotesInterface", new BraidService(appServiceHub))
                .withPort((Integer) serviceHub.getAppContext().getConfig().get("braid"))
                .withHttpServerOptions(new HttpServerOptions().setSsl(false))
                .bootstrapBraid(appServiceHub, asyncResult -> System.out.println(asyncResult.result()));
    }
}

File: kotlin-source/src/main/kotlin/net/corda/accord/BraidServer.kt
package net.corda.accord

import io.bluebank.braid.corda.BraidConfig
import io.vertx.core.AsyncResult
import io.vertx.core.Handler
import io.vertx.core.http.HttpServerOptions
import net.corda.accord.flow.PromissoryNoteIssueFlow
import net.corda.accord.flow.PromissoryNoteIssueJSONFlow
import net.corda.core.node.AppServiceHub
import net.corda.core.node.services.CordaService
import net.corda.core.serialization.SingletonSerializeAsToken


@CordaService
class BraidServer(appServiceHub: AppServiceHub) : SingletonSerializeAsToken() {

    init {

        val braidHandler: Handler<AsyncResult<String>> = Handler { asyncResult -> System.out.println(asyncResult.result()) }

        BraidConfig()
                .withFlow("PromissoryNoteIssueFlow", PromissoryNoteIssueFlow::class.java)
                .withFlow("PromissoryNoteIssueJSONFlow", PromissoryNoteIssueJSONFlow.InitiatorFlow::class.java)
                .withService("PromissoryNotesInterface", BraidService(appServiceHub))
                .withPort(appServiceHub.getAppContext().config.get("braid") as Int)
                .withHttpServerOptions(HttpServerOptions().setSsl(false))
                .bootstrapBraid(appServiceHub, braidHandler)
    }
}
